### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by asbiin**
@zepfietje Would you accept some tests?

### CODE REVIEW

**Comment by zepfietje** [comment]
Sure, @asbiin!

Is this ready for review?

**Comment by asbiin** [comment]
> Is this ready for review?

yes!

File: tests/PirschTest.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+it('can track request', function () {
+    config(['pirsch.token' => 'test_token']);
+    Http::fake([
+        'https://api.pirsch.io/api/v1/hit' => Http::response(),
+    ]);
+
+    Pirsch::track();
+    $this->get('/');
+
+    Http::assertSent(function (Request $request) {
```
Comment (zepfietje) [inline tests/PirschTest.php]:
```suggestion
    Http::assertSent(function (Request $request): bool {
```

File: tests/PirschTest.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Illuminate\Http\Client\Request;
+use Illuminate\Support\Facades\Http;
+use Pirsch\Facades\Pirsch;
+
+it('can track request', function () {
```
Comment (zepfietje) [inline tests/PirschTest.php]:
```suggestion
it('can track a request', function () {
```

File: tests/TrackPageviewTest.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    Route::middleware(TrackPageview::class)
+        ->get('/', fn () => redirect('/home'));
+
+    $this->get('/');
+
+    Pirsch::shouldNotHaveBeenCalled();
+    expect(true)->toBeTrue();
+});
+
+it('skips livewire', function () {
```
Comment (zepfietje) [inline tests/TrackPageviewTest.php]:
```suggestion
it('skips Livewire', function () {
```

File: tests/TrackPageviewTest.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    Route::middleware(TrackPageview::class)
+        ->get('/', fn () => 'Hello World');
+
+    $this->get('/', ['X-Livewire' => 'true']);
+
+    Pirsch::shouldNotHaveBeenCalled();
+    expect(true)->toBeTrue();
+});
+
+it('skips telescope', function () {
```
Comment (zepfietje) [inline tests/TrackPageviewTest.php]:
```suggestion
it('skips Telescope', function () {
```

File: tests/TrackPageviewTest.php [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+it('handles request', function () {
+    Pirsch::shouldReceive('track')
+        ->once();
+
+    Route::middleware(TrackPageview::class)
+        ->get('/', fn () => 'Hello World');
+
+    $this->get('/');
+
+    expect(true)->toBeTrue();
```
Comment (zepfietje) [inline tests/TrackPageviewTest.php]: Isn't it cleaner to use `throwsNoExceptions()` instead?
https://pestphp.com/docs/exceptions

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: tests/TrackPageviewTest.php (hunk 5 above)
Comment (asbiin) [reply tests/TrackPageviewTest.php]: As you wish!

Comment (zepfietje) [reply tests/TrackPageviewTest.php]: If I'm correct and the behavior stays the same for these tests, could you please update those? :)

Comment (asbiin) [reply tests/TrackPageviewTest.php]: Actually, those tests already have an assertion, and with `throwsNoExceptions` it lead to this error:
```
 RISKY  Tests\TrackPageviewTest > it handles request                                                                                                                                        
  This test is not expected to perform assertions but performed 1 assertion
```
... so I just removed the `except..`

**Comment by zepfietje** [comment]
Thanks, @asbiin!

**Pull request merged**